database/migrations/2021_02_15_184651_create_airtable_ids_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateAirtableIdsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('airtable_ids', function(Blueprint $table) {
+            $table->string('airtable_id')->unique();
+            $table->unsignedBigInteger('activity_instance_id');
```

Comment:
Can also be unique!

Reply:
Do you mean that the activity_instance)id can be unique or the combo of both?

Reply:
Both must be independently unique, so just adding the unique() method to the actiivty_instance_id would be great :) !